mer_verify_kernel_config

```diff
@@ -142,12 +200,12 @@ CONFIG_CGROUPS			y       # systemd: http://cgit.freedesktop.org/systemd/systemd/
 CONFIG_CGROUP_FREEZER		y,!	# systemd (optional): http://0pointer.de/blog/projects/cgroups-vs-cgroups.html
 CONFIG_CGROUP_DEVICE		y,!	# systemd (optional): http://0pointer.de/blog/projects/cgroups-vs-cgroups.html
 CONFIG_CGROUP_CPUACCT		y,!	# systemd (optional): http://0pointer.de/blog/projects/cgroups-vs-cgroups.html
-CONFIG_CGROUP_MEM_RES_CTLR	y,!	# systemd (optional): http://0pointer.de/blog/projects/cgroups-vs-cgroups.html, ignore if kernel version >= 3.10
-CONFIG_CGROUP_MEM_RES_CTLR_SWAP	y,!	# systemd (optional): http://0pointer.de/blog/projects/cgroups-vs-cgroups.html, ignore if kernel version >= 3.10
-CONFIG_CGROUP_MEM_RES_CTLR_KMEM	y,!	# systemd (optional): http://0pointer.de/blog/projects/cgroups-vs-cgroups.html, ignore if kernel version >= 3.10
-CONFIG_MEMCG			y,!	# systemd (optional, but recommended): http://0pointer.de/blog/projects/cgroups-vs-cgroups.html, ignore if kernel version < 3.10
-CONFIG_MEMCG_SWAP		y,!	# systemd (optional, but recommended): http://0pointer.de/blog/projects/cgroups-vs-cgroups.html, ignore if kernel version < 3.10
-CONFIG_MEMCG_KMEM		y,!	# systemd (optional, but recommended): http://0pointer.de/blog/projects/cgroups-vs-cgroups.html, ignore if kernel version < 3.10
+CONFIG_CGROUP_MEM_RES_CTLR	y,!,<=3.10	# systemd (optional): http://0pointer.de/blog/projects/cgroups-vs-cgroups.html, only valid if kernel version <= 3.10
```

Comment:
Actually the correct version numbering for these would be CONFIG_CGROUP_MEM_RES_CTLR* <=3.5 and CONFIG_MEMCG >= 3.6. For reference https://cateee.net/lkddb/web-lkddb/CGROUP_MEM_RES_CTLR.html and https://cateee.net/lkddb/web-lkddb/MEMCG.html. Although Android only uses 3.4 and 3.10 so the version berween are not relevant but still would be more correct to use 3.5 and 3.6.